cou/cli.py

```diff
@@ -291,4 +292,6 @@ def entrypoint() -> None:
     finally:
         if args.command == "upgrade":
             loop.run_until_complete(run_post_upgrade_sanity_check(args))
+        if log_file:
```

Comment:
@jneo8 I remember you added `log_file = None` before the try block; what happened to that?  That will be required here.

Reply:
Maybe something lost when rebasing, let me change it.

Reply:
Updated, resolved